packages/block-editor/src/components/iframe/index.js

```diff
@@ -310,7 +318,7 @@ function Iframe( {
 			'--wp-block-editor-iframe-zoom-out-scale',
 			scale === 'default'
 				? Math.min( containerWidth, maxWidth ) /
-						prevContainerWidthRef.current
+						( prevContainerWidthRef.current || containerWidth )
```

Comment:
Do we still need this?

Reply:
Right, this is no longer requires as zoom out is reset on component unmount, and `prevContainerWidthRef.current` is always set when component is mounted.

Removed in e6303353ebd6c4562dc7fb064e6f90446b25f31f